components/eamxx/cime_config/namelist_defaults_scream.xml

```diff
@@ -5,8 +5,9 @@
 The file that defines all parameter input files for SCREAM. This file
 will be processed by SCREAM's buildnml script and produces the
 $case/namelist_scream.xml which is the fully resolved/processed
-info from this file. We recommend users use atm-config-chg for
-tweaking their $case/namelist_scream.xml file.
+info from this file. We highly recommend users use atmchange for
+tweaking their $case/namelist_scream.xml file. Direct edits will
+be lost if SCREAM_HACK_XML is not enabled.
```

Comment:
Note: in case you decide to change the XML var name, it should be replaced here too.